playbooks/adhoc/atomic_openshift_tutorial_reset.yml

```diff
@@ -0,0 +1,68 @@
+# This deletes *ALL* Docker images, and uninstalls OpenShift and
+# Atomic Enterprise RPMs.  It is primarily intended for use
+# with the tutorial as well as for developers to reset state.
+
+- hosts:
+    - OSEv3:children
+
+  sudo: yes
+
+  tasks:
+    - service: name={{ item }} state=stopped
+      with_items:
+        - openshift-master
+        - openshift-node
+        - openvswitch
+        - atomic-enterprise-master
+        - atomic-enterprise-node
+
+    - yum: name={{ item }} state=absent
+      with_items:
+        - openvswitch
+        - atomic-enterprise
+        - atomic-enterprise-master
+        - atomic-enterprise-node
+        - atomic-enterprise-sdn-ovs
+        - tuned-profiles-atomic-enterprise-node
+        - openshift
+        - openshift-master
+        - openshift-node
+        - openshift-sdn-ovs
+        - tuned-profiles-openshift-node
```

Comment:
May as well add `origin` `origin-master` `origin-node` `origin-sdn-ovs` and `tuned-profiles-origin-node`